.travis.yml

```diff
@@ -1,6 +1,16 @@
 language: node_js
 node_js:
-  - "0.10"
-  - "0.11"
-  - "0.12"
-
+  - "10.15.3"
```

Comment:
actually i recommend just setting this to "node" (assuming you don't need a specific version) because then it will always use latest stable. otherwise .travis.yml needs to be updated every time node stable updates. https://docs-staging.travis-ci.com/user/languages/javascript-with-nodejs/

Reply:
done, now using "node"